src/components/date-input/error/index.njk

```diff
@@ -11,16 +11,16 @@ ignore_in_sitemap: true
   namePrefix: "dob",
```

Comment:
We may want to change 'dob' to something more inline with the new password content.